babel.config.js

```diff
@@ -51,25 +51,25 @@ module.exports = function(api) {
       isTestEnv && 'babel-plugin-dynamic-import-node',
       '@babel/plugin-transform-destructuring',
       [
-        '@babel/plugin-proposal-class-properties',
+        '@babel/plugin-transform-class-properties',
```

Comment:
These fixes are from this issue: https://github.com/rails/rails/issues/48372#issue-1737187584